code/game/objects/items/his_grace.dm

```diff
@@ -20,14 +20,71 @@
 	hitsound = 'sound/items/weapons/smash.ogg'
 	drop_sound = 'sound/items/handling/toolbox/toolbox_drop.ogg'
 	pickup_sound = 'sound/items/handling/toolbox/toolbox_pickup.ogg'
+	gender = MALE
 	var/awakened = FALSE
 	var/bloodthirst = HIS_GRACE_SATIATED
 	var/prev_bloodthirst = HIS_GRACE_SATIATED
 	var/force_bonus = 0
 	var/ascended = FALSE
 	var/victims_needed = 25
 	var/ascend_bonus = 15
+	var/pronouns = list(
+		"his",
+		"His",
+		"Him",
+		"he",
+		"He",
+		"himself",
+	)
+
 
+/obj/item/his_grace/item_interaction(mob/living/user, obj/item/tool, list/modifiers)
+	if(istype(tool,/obj/item/slimepotion/genderchange))
+		var/turf/T = get_turf(src)
+		var/mob/living/holder
+		if(istype(loc, /mob/living))
+			holder = loc
+		if(gender == MALE)
+			pronouns = list(
+				"her",
+				"Her",
+				"Her",
+				"she",
+				"She",
+				"herself",
+			)
```

Comment:
Why are we bothering with this if we're changing the object's gender, any of the pronoun helpers would work